src/main/java/org/javamoney/moneta/internal/convert/USFederalReserveRateReadingHandler.java

```diff
@@ -0,0 +1,179 @@
+package org.javamoney.moneta.internal.convert;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
+import java.util.Collections;
+import java.util.Currency;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.money.CurrencyUnit;
+import javax.money.Monetary;
+import javax.money.MonetaryException;
+import javax.money.convert.ConversionContextBuilder;
+import javax.money.convert.ExchangeRate;
+import javax.money.convert.ProviderContext;
+import javax.money.convert.RateType;
+
+import org.javamoney.moneta.convert.ExchangeRateBuilder;
+import org.javamoney.moneta.spi.DefaultNumberValue;
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+/**
+ * SAX Event Handler that reads the quotes.
+ * <p>
+ * RDF format: <item rdf:about="http://www.federalreserve.gov/releases/H10#16"> <title>US: H10 0.7100 2015-08-31 FRB
+ * Australia Dollar (USD per AUD)</title> <link>http://www.federalreserve.gov/releases/H10#16</link>
+ * <description>Australia Dollar (USD per AUD)</description> <dc:date>2015-08-31T12:00:00-05:00</dc:date>
+ * <dc:language>en</dc:language> <dc:creator>FRB</dc:creator> <cb:statistics> <cb:country>US</cb:country>
+ * <cb:institutionAbbrev>FRB</cb:institutionAbbrev> <cb:otherStatistic> <cb:value decimals="4" unit_mult="1"
+ * units="Currency:_Per_AUD">0.7100</cb:value> <cb:topic>H10</cb:topic> <cb:coverage>Australia Dollar (USD per
+ * AUD)</cb:coverage> <cb:observationPeriod frequency="business">2015-08-31</cb:observationPeriod> <cb:dataType/>
+ * </cb:otherStatistic> </cb:statistics> </item>
+ */
+class USFederalReserveRateReadingHandler extends DefaultHandler {
+    private LocalDate localDate;
+    private String currencyCode;
+    private String description;
+
+    private boolean dcDateNode = false;
+    private boolean cbValueNode = false;
+    private boolean descriptionNode = false;
+
+    private final Map<LocalDate, Map<String, ExchangeRate>> historicRates;
+
+    private final ProviderContext context;
+
+    private static final Pattern unitsPattern = Pattern.compile("^Currency:_Per_(\\w{3})$");
+
+    private static final Map<String, CurrencyUnit> CURRENCIES_BY_NAME;
+
+    static {
+        Map<String, CurrencyUnit> _currencyiesByName = new HashMap<>();
```

Comment:
Yes, but proposing it to https://github.com/JavaMoney/javamoney-lib/tree/master/exchange as "exchange-rate-usfed" or similar has a point. @jdyer1 would you consider doing a new pull-requests there (after which we may close this, let's leave it open as reminder for now)
That place requires no OCA or JCP membership, anybody can propose something there;-)